### PR TITLE
0131 release notes in doc

### PR DESCRIPTION
**Pull request by TeddyCr**
### Describe your changes :
Added release notes for 0.13.1 in documentations

#
### Type of change :

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by ayush-shah — APPROVED** [review]

File: openmetadata-docs/content/overview/releases/latest-release.md [hunk 1]
```diff
@@ -4,157 +4,21 @@ slug: /overview/latest-release
...
-
-
-## Thanks to our Contributors
-We are thankful for the overwhelming feedback and support we received from our community. We are grateful to the following community members for their code contributions:
-
-- Pedro Sereno for capturing Metabase lineage with SQL lineage.
-- Francisco J. Jurado Moreno — for working on caching tables when ingesting database usage; optimizing Redshift usage query; Datalake connector performance; non-scheduled workflows; and to add a comment to tag all OpenMetadata-related queries.
-- Nihar Doshi — for helping with cleaning Atlas, Amundsen, and metadata_rest; and checking MSSQL with windows authentication.
-
-Thanks to Abcabhishek, Bleachzk, Bryson Edwards, Daniel, Fikrifikar, geoHeil, Laila Patel, Nicolas Parot Alvarez, Nilesh Khatri, Pauline Tolstova, Pedro Sereno, Preeti Jain, Sam Firke, Samuel Stuetz, Sidharth Reddy Pallerla, Taurus-Le, TheFu527, Tomislav Sabados, Trillhaa, Upen Bendre for creating issues in GitHub. Thanks to Sergey Stepanov, Zlgonzalez for your feedback and suggestions.
```
Comment (ayush-shah) [inline openmetadata-docs/content/overview/releases/latest-release.md:160]: do we have a list of contributors, if so, we can add those in the latest-release?

Comment (ayush-shah) [reply openmetadata-docs/content/overview/releases/latest-release.md:160]: @TeddyCr ^

Comment (ayush-shah) [reply openmetadata-docs/content/overview/releases/latest-release.md:160]: New Contributors
@07Himank made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9006
@dhruvinmaniar123 made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9147
@talalio made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9174
@karanh37 made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9307

**Review by ayush-shah — DISMISSED** [review]
Here is the list of new contributors

New Contributors:
@07Himank made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9006
@dhruvinmaniar123 made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9147
@talalio made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9174
@karanh37 made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9307

**Review by pmbrull — APPROVED** [review]

**Pull request merged**